docs/pluggable_streams.rst

```diff
@@ -205,7 +211,7 @@ Below is a sample table config used to create a realtime table with Kafka Partit
       "loadMode": "MMAP",
       "streamConfigs": {
         "streamType": "kafka",
-        "stream.kafka.consumer.type": "simple",
+        "stream.kafka.consumer.type": "LowLevel",
```

Comment:
Change line 124 from `lowlevel` to `LowLevel` as well? 

It doesn't hurt as is since ultimately they are all converted to uppercase, but would be nice to make the configs more consistent and avoid confusion.

Reply:
Agreed, changed.